### PR TITLE
Add agentImageRef parameter to support external image refs

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

**Comment by yanmxa** [comment]
@Refs: https://github.com/stolostron/backlog/issues/24259

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,23 @@
 SHELL :=/bin/bash
 
+REGISTRY ?= quay.io/open-cluster-management-hub-of-hubs
+IMAGE_TAG ?= latest
+/:latest
```
Comment (morvencao) [inline Makefile:5]: what's this?

Comment (yanmxa) [reply Makefile]: the `/:latest` is careless mistake 😓

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**